src/overlays/gamestates/ovl_daytelop/z_daytelop.c

```diff
@@ -5,10 +5,11 @@
  */
 
 #include "z_daytelop.h"
+#include "z64save.h"
 #include "z64shrink_window.h"
-#include "z64view.h"
```

Comment:
Why is view.h is moved to the header but not shrink_window.h?

Reply:
Because `shrink_window.h` is not needed by the header, but the `DayTelopState` struct requires the `View` struct